minutes/2021-Sept.md

```diff
@@ -0,0 +1,64 @@
+**Present:** Peter Cock, Chris Fields, Heather Wiencko, Nomi Harris, Bastian Greshake-Tzovaras, Hilmar Lapp
+**Apologies:** Malvika Sharan (board member), Hilyatuz Zahroh (board candidate)
+
+**Also attending:** Caleb Kibet (board candidate), Yo Yehudi, Pablo Riesgo Ferreiro
+
+*Agenda:* Link Needed
```

Comment:
blog post before the meeting is [here](https://www.open-bio.org/2021/09/11/obf-public-board-meeting-2021-09-21/); blog post after the meeting is [here](https://www.open-bio.org/2021/10/06/two-new-members-elected-to-obf-board/)

lmk if you need the gdoc with internal notes from the meeting

Reply:
Rather than a link (for a non-static GDoc), can we just paste in the text?

Reply:
@peterjc text as the standard makes sense since this would be a static document.

Reply:
```suggestion

**Agenda:**
Please see the [Blog Post for agenda points](https://www.open-bio.org/2021/09/11/obf-public-board-meeting-2021-09-21/)
```

Reply:
Consensus is to copy the agenda bullet points here (the board elections and voting on policy changes).